src/Streamly/Internal/Data/Stream/SVar/Generate.hs

```diff
@@ -160,8 +160,7 @@ fromStreamVar sv = K.MkStream $ \st yld sng stp -> do
             StopBy -> do
                 sid <- liftIO $ readIORef (svarStopBy sv)
                 return $ tid == sid
-
-#if __GLASGOW_HASKELL__ < 810
+#if __GLASGOW_HASKELL__ > 810
```

Comment:
Remove this conditional.

Reply:
CI Fails: https://github.com/composewell/streamly/actions/runs/3214056803/jobs/5254165605

Reply:
after removing the conditional.